### ISSUE TITLE
indentRainbow.colors size should be flexible

### ISSUE THREAD

**Issue by Tyriar** [oderwat/vscode-indent-rainbow#6]
I'd love to be able to specify 1+ colors and have them loop around as expected, right now 4 colors are enforced.

![image](https://cloud.githubusercontent.com/assets/2193314/15261192/cadff072-190f-11e6-8348-be9abdbfefdf.png)

Specifically I want to implement indent guides of a slightly transparent and completely transparent. This is possible right now using:

``` json
  "indentRainbow.colors": [
    "rgba(255,255,255,0.05)",
    "rgba(0,0,0,0)",
    "rgba(255,255,255,0.05)",
    "rgba(0,0,0,0)"
  ]
```

But should be using:

``` json
  "indentRainbow.colors": [
    "rgba(255,255,255,0.05)",
    "rgba(0,0,0,0)"
  ]
```

**Comment by oderwat** [comment]
Well.. there are 5 colors used such that the "error" color is the fifth defined one and the cycle has 4 colors. I think one could make the error color an extra color which can be "empty" and leave the remaining colors cycling automatically. I'd say a PR is welcome. Not sure if I want to add that as I would never use it and what you want us already possible.

**Comment by Tyriar** [comment]
Oh there's an error color? Does that show up on lint errors?

**Comment by oderwat** [comment]
It shows up on non multiples of the set spacing if used with spaces. Like if you have 3 spaces in a 2 space setup or 5 in a 4 space setup.

Changing the colors to be dynamically assignable would need a major rewrite of the code as there are currently 5 decorations hardcoded.

**Comment by oderwat** [comment]
Closed by #13
